src/Projections/Reunion/Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.cs

```diff
@@ -17,19 +23,53 @@ public DispatcherQueueSynchronizationContext(DispatcherQueue dispatcherQueue)
             m_dispatcherQueue = dispatcherQueue;
         }
 
-        public override void Post(SendOrPostCallback d, object state)
+        /// <inheritdoc/>
+        public override unsafe void Post(SendOrPostCallback d, object? state)
         {
-            if (d == null)
-                throw new ArgumentNullException(nameof(d));
+            if (d is null)
+            {
+                static void ThrowArgumentNullException()
+                {
+                    throw new ArgumentNullException(nameof(d));
+                }
+
+                ThrowArgumentNullException();
```

Comment:
```suggestion
                throw new ArgumentNullException(nameof(d));
```
This throw helper micro-optimization is not worth it given everything else that this method does.

Reply:
Updated in e95e80e3b39ce94bc0b25e8d0692cb8625e42e10 🙂